File: frontend/quiz/src/seleniumTests/Pages/ModalLoginPage.js
import { getElementById } from "./Utility";
import Element2 from "./Element";

class ModalLoginPage extends Element2 {
  get closeModalBtn() {
    return getElementById(this.driver, "close");
  }
  async getText() {
    return await getElementById(this.driver, "modalText").then(el =>
      el.getText()
    );
  }
  async isVisible() {
    return super.isVisibleById("close");
  }
}

export default ModalLoginPage;

File: backend/selenium/ModalLogInPage.py
from Utility import getElementById, isVisibleById
from Page import Page


class ModalLogInPage (Page):
    def closeModalBtn(self):
        return getElementById(self.driver, "close")

    def getText(self):
        return getElementById(self.driver, "modalText").text

    def isVisible(self):
        return isVisibleById(self.driver,  "close")
